SwiftWisdom/SwiftWisdom/Core/Version/Version.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  Version.swift
+//  bmap
+//
+//  Created by Logan Wright on 11/30/15.
+//  Copyright © 2015 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+public struct Version {
+    public private(set) var major: Int = 0
+    public private(set) var minor: Int = 0
+    public private(set) var patch: Int = 0
+    
+    init?(string: String) {
+        let components = string
+            .componentsSeparatedByString(".")
```

Comment:
Not explicitly, but we should look for a nice way to allow for newline situations.  This one is possibly a bit of a stretch, but often it's a good way to keep lines compact.